.github/actions/github-dependabot-check/action.yml

```diff
@@ -0,0 +1,44 @@
+name: Dependabot check
+description: Checks if author is dependabot and if specific label is added to PR
+
+inputs:
+  gh-token:
+    description: 'GitHub token with content read permission'
+    required: true
+  label:
+    description: 'Check if the pull request contains this label'
+    required: true
+
+outputs:
+  result:
+    description: "Result of the check"
+    value: ${{ steps.check.outputs.result }}
+
+runs:
+  using: composite
+  steps:
+    - name: Check
+      id: check
+      env:
+        GH_TOKEN: ${{ inputs.gh-token }}
+        SHA: ${{ github.event.pull_request.head.sha || github.sha }}
+        FOUND_LABEL: ${{ inputs.label }}
+        GITHUB_EVENT_NAME: ${{ github.event_name }}
```

Comment:
this one is already mapped in bash, it should not need to be redefined (like GITHUB_ACTOR) https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables